client/app/components/Parameters.less

```diff
@@ -20,27 +20,43 @@
   }
 
   &.parameter-dragged {
-    box-shadow:  0 4px 9px -3px rgba(102, 136, 153, 0.15);
+    box-shadow: 0 4px 9px -3px rgba(102, 136, 153, 0.15);
   }
 }
 
 .parameter-heading {
   display: flex;
   align-items: center;
+  justify-content: stretch;
   padding-bottom: 4px;
 
   label {
+    flex: 1 1 auto;
     margin-bottom: 1px;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    min-width: 100%;
-    max-width: 195px;
-    white-space: nowrap;
-
-    .parameter-block[data-editable] & {
-      min-width: calc(100% - 27px); // make room for settings button
-      max-width: 195px - 27px;
```

Comment:
It may be trickier to reach the same Label behavior for parameters:

**Before**
![image](https://user-images.githubusercontent.com/3356951/73593812-ce263a00-44e6-11ea-9938-0c3c66717f76.png)
**After**
![image](https://user-images.githubusercontent.com/3356951/73593821-dda58300-44e6-11ea-94bf-214972a7faf1.png)

The idea is to keep it with a flexible width, if the label itself has more text and the width is not along it's limit, the whole parameter can take up more space :slightly_smiling_face:.  You couldn't get it to work with the previous css or you just wanted to move it to flex?

Reply:
I wanted to make it flex (and get rid of `calc`). Thank you for pointing, I'll check this once more :+1: